src/styles.css

```diff
@@ -311,3 +310,84 @@ ul {
 .languages_list {
   flex-wrap: wrap;
 }
+
+.footer {
+  background-image: url("../media/images/contactme.png");
+  height: 100vh;
+  background-repeat: no-repeat;
+  background-position: 100% 85%;
+  display: grid;
+  grid-template-rows: 85% 7% 5%;
+}
+
+form > ul {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+}
+
+form > ul > li > input::placeholder { /* Chrome, Firefox, Opera, Safari 10.1+ */
+  color: #6b778c;
+  opacity: 1; /* Firefox */
+}
+
```

Comment:
- [ ] The text inside of inputs aligned strongly left. It will be better to add `padding` to make closer to Figma design:

> Your inputs
<img width="276" alt="Screen Shot 2022-01-12 at 11 25 51" src="https://user-images.githubusercontent.com/49064106/149082564-c6736b0e-9343-48a8-9b46-78543378cbec.png">

> Figma template
<img width="255" alt="Screen Shot 2022-01-12 at 11 26 35" src="https://user-images.githubusercontent.com/49064106/149082637-24b02483-ffc7-4bc5-b9fd-4e588da4732e.png">